relayer/application.ts

```diff
@@ -56,10 +56,10 @@ export type FetchaVaasOpts = {
 };
 
 export const defaultWormholeRpcs = {
-  [Environment.MAINNET]: ["https://api.wormscan.io"],
+  [Environment.MAINNET]: ["https://api.wormholescan.io"],
   [Environment.TESTNET]: [
     "https://wormhole-v2-testnet-api.certus.one",
-    "https://api.testnet.wormscan.io",
+    "https://api.testnet.wormholescan.io",
```

Comment:
changed because rate limiting in  testnet.wormholescan.io is 1000/min vs wormscan that is 100/min

Reply:
does this apply for mainnet as well?

Reply:
No, both domains have the same limit: 1000, but figured we should just update it